datastew/repository/weaviate.py

```diff
@@ -62,7 +64,35 @@ def _create_schema_if_not_exists(self, schema):
         except Exception as e:
             raise RuntimeError(f"Failed to check/create schema for {class_name}: {e}")
 
-    def store_all(self, model_object_instances):
+    def import_data_dictionary(self, data_dictionary: DataDictionarySource, terminology_name: str, embedding_model: Optional[EmbeddingModel] = None):
+        terminology = Terminology(terminology_name, terminology_name)
+        self.store(terminology)
+        data_frame = data_dictionary.to_dataframe()
```

Comment:
Same here - would also make sense to handle this defensively with a try/catch block